requirements/extra.txt

```diff
@@ -4,7 +4,7 @@ neptune-client>=0.4.109
 comet-ml>=3.1.12
 mlflow>=1.0.0
 test_tube>=0.7.5
-wandb>=0.8.21
+wandb>=0.8.21, <0.10.0  # higher version mess with system paths
```

Comment:
Just a heads up, all versions do something to `sys.path`, but 0.10.0 adds even more stuff.